src/main/java/TaskManager.java

```diff
@@ -0,0 +1,126 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    protected ArrayList<Task> tasks;
+    protected DukeInterface dukeUI;
+    private int numTasks;
+    private int numComplete;
+
+    private final String ADD_TASK_VALID_MSG = "Chomp-chomp! I've added this new task [\uD83D\uDCDD]:";
+    private final String SET_TASK_COMPLETE_VALID_MSG = "Burrrp! I've marked this task as done [\u2705]:";
+    private final String SET_TASK_COMPLETE_INVALID_MSG = "Hold up! This task is already marked done [\u2705]:";
+    private final String PRINT_TASK_VALID_MSG = "Ahh! Here are the tasks in your list [\uD83D\uDCC5]:";
+    private final String PRINT_TASK_INVALID_MSG = "Sorry, but there are currently no tasks added!";
+    private final String GENERIC_PROMPT_MSG = "You may choose to add/complete another task.";
+
+    public TaskManager() {
+        tasks = new ArrayList<Task>();
+        dukeUI = new DukeInterface();
+        numTasks = 0;
+        numComplete = 0;
+    }
+
+    public void addTask(String[] taskInfo) {
+        Task newTask;
+        String taskType = taskInfo[0];
+        String taskDescription = "";
+        String taskDateField = "";
+
+        switch (taskType) {
+        case "todo":
+            for (int i = 1; i < taskInfo.length; i++) {
+                taskDescription += taskInfo[i] + " ";
+            }
+
+            taskDescription = taskDescription.trim();
+            newTask = new ToDo(taskDescription);
+            this.tasks.add(newTask);
+            break;
+
+        case "deadline":
+            int i = 1;
```

Comment:
I understand you want to use this "i" in another for loop, but this leads to the first for loop no following the coding standard.